### PR TITLE
Milestone-3-style

### PR DESCRIPTION
**Pull request by nriqu322**
Style added to Display and Button components.

- Background color default orange for button and grey
- Set 700px to calculator width
- Set 100px height to button
- Set 25% width to button
- Add color and wide as props to the Button component

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Display from './Display';
 
 function App() {
   return (
-    <div>
+    <div className="calculator">
```
Comment (Oghenebrume50) [inline src/components/App.js:7]: App component needs to use flex display 
see this

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -3,33 +3,33 @@ import Button from './Button';
...
+      <Button name="2" color="#e0e0e0" />
+      <Button name="3" color="#e0e0e0" />
       <Button name="+" />
     </div>
-    <div className="group5">
-      <Button name="0" />
-      <Button name="." />
+    <div className="group">
+      <Button name="0" wide color="#e0e0e0" />
+      <Button name="." color="#e0e0e0" />
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js:32]: When you set default props for a component you can avoid passing that prop when it is not needed that is if it should use the default value anyway

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @nriqu322 good job working on this, this looks really good :+1:  however I have outlined some few changes below, **please take a look and request for another review when done** 

**RAPHAEL NORIODE** :mask: STAY_SAFE

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (nriqu322) [reply src/components/ButtonPanel.js:32]: In the requirements the default value is orange : color: '#f5913e', so there I'm passing the light gray color for the rest of the buttons.

File: src/components/App.js (hunk 1 above)
Comment (nriqu322) [reply src/components/App.js:7]: done

**Review by bellom — APPROVED** [review]
## Hi @nriqu322  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by nriqu322** [comment]
thanks.

**Pull request merged**